handout/modules/lazy-loading-module.md

```diff
@@ -0,0 +1,152 @@
+# Lazy Loading a Module
+
+Another advantage of using modules to group related pieces of functionality of our application, is the ability to load those pieces on demand. Lazy loading modules will help us decrease the startup time of our application because we don't need to load everything at once, we only need to load what the user expects to see at the beginning and then conditionally load other modules as the user navigates to different sections. As you can imagine, there's a relationship between lazy loading and routing.
```

Comment:
- remove the comma from `functionality of our application, is the`
- change `Lazy loading modules will help us decrease` to `Lazy loading modules helps us decrease`
- break the sentence `Lazy loading modules will help us decrease the startup time of our application because we don't need to load everything at once, we only need to load what the user expects to see at the beginning and then conditionally load other modules as the user navigates to different sections.` into `...decrease the startup time.  With lazy loading our application does not need to load everything at once, it only needs to load what the user expects to see when the app first loads.  Modules that are lazily loaded will only be loaded when the user navigates to their routes.`